services/mailservice/inbucket.go

```diff
@@ -57,18 +58,19 @@ func GetMailBox(email string) (results JSONMessageHeaderInbucket, err error) {
 	parsedEmail := ParseEmail(email)
 
 	url := fmt.Sprintf("%s%s%s", getInbucketHost(), INBUCKET_API, parsedEmail)
-	req, err := http.NewRequest("GET", url, nil)
+	resp, err := http.Get(url)
+
 	if err != nil {
 		return nil, err
 	}
 
-	client := &http.Client{}
+	defer func() {
+		// To reuse same tcp connection
```

Comment:
Let's remove the comments. The draining of the body is done in a lot of places.

Reply:
done